modules/ROOT/pages/vantagecloud-lake-demos-visual-studio-code.adoc

```diff
@@ -0,0 +1,111 @@
+= Run VantageCloud Lake Demos from a Jupyter notebook in Visual Studio Code
+:experimental:
+:page-author: Janeth Graziani
+:page-email: Janeth.graziani@teradata.com
+:page-revdate: January 11, 2024
+:description: Learn how to run VantageCloud Lake Demos using Jupyter notebooks in Visual Studio Code. 
+:keywords: data warehouses, compute storage separation, teradata, vantage, cloud data platform, business intelligence, enterprise analytics, jupyter, teradatasql, ipython-sql, teradatasqlalchemy, vantagecloud, vantagecloud lake, public internet, visual studio code, IDE, data analytics, data science
+
+== Overview
+Visual Studio Code is a popular open-source code editor compatible with Windows, MacOs, and Linux. Developers use this Integrated Development Environment (IDE) for coding, debugging, building, and deploying applications. In this quickstart guide, we launch VantageCloud Lake Jupyter notebook demos within Visual Studio Code. 
+
+image::vscode.png[vscode.png,align="center", width=80%]
+
+== Prerequisites
+Before you begin, ensure you have the following prerequisites in place:
+
+* https://www.docker.com/products/docker-desktop/[Docker Desktop] installed
+* https://git-scm.com/book/en/v2/Getting-Started-Installing-Git[Git] installed
+** Required to download git repo from https://github.com/Teradata/lake-demos.git
+* https://code.visualstudio.com/download[Visual Studio Code] installed
+* A Teradata VantageCloud Lake account with organization URL and login details from the Teradata welcome letter
+
+
+== Clone VantageCloud Lake Demo repository 
+Begin by cloning the GitHub repository and navigating to the project directory:
+----
+git clone https://github.com/Teradata/lake-demos.git
+cd lake-demos
+----
+== Mount Docker files 
+To launch Teradata VantageCloud Lake demos, we need the https://hub.docker.com/r/teradata/jupyterlab-extensions[Jupyter extension for Docker]. This extension bundles Teradata SQL kernel, Teradata Python, R libraries, drivers and Teradata extensions for seamless interaction with Teradata databases. 
+----
+docker pull teradata/jupyterlab-extensions
+----
+Next, start a container and bind it to the existing lake-demos directory. Replace */path/to/your/directory* with your actual directory:
```

Comment:
The user is already in the right directory so they can use `PWD` to reference it. It's O/S dependent so please check this comment on how to implement it: https://github.com/Teradata/quickstarts/pull/150/files#r1448885132